README.md

```diff
@@ -198,17 +198,25 @@ $ docker image pull confluentinc/cp-kafka:5.4.3
 $ ulimit -n 1024
 ```
 
-### Building Feathub Project
-
-1. Use the following command to generate protobuf files. You can follow this
+4. Make sure protoc 3.17 is installed in your development environment. You can
+   follow this
    [README](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation)
    to install protoc.
 
 ```bash
-$ protoc -I=python/feathub/common/protobuf/ --python_out=python/feathub/common/protobuf/ value.proto
+$ protoc --version
```

Comment:
It seems simpler to just specify the proto version we need. And we don't need to explicitly specify the requirement here if we can dynamically check proto version and throw error with proper message when we run `python -m pip install ./python`.